widgets/blueprints/src/widget.tsx

```diff
@@ -86,11 +93,17 @@ Stage.defineWidget<unknown, unknown, BlueprintsWidgetConfiguration>({
 
     fetchData(widget, toolbox, params) {
         const result = {};
+
         return toolbox
             .getManager()
-            .doGet(
-                '/blueprints?_include=id,updated_at,created_at,description,created_by,visibility,main_file_name,state,error',
-                { params }
+            .doPost(
+                '/searches/blueprints?_include=id,updated_at,created_at,description,created_by,visibility,main_file_name,state,error',
+                {
+                    params,
```

Comment:
We can make it cleaner at once:
```suggestion
                '/searches/blueprints',
                {
                    _include: 'id,updated_at,created_at,description,created_by,visibility,main_file_name,state,error',
                    ...params,
```